doc/release-notes/6.7-release-notes.md

```diff
@@ -0,0 +1,663 @@
+
+### New Metadata Field Type: String
+
+The "string" type has been added as a new field type for metadata fields.
+
+In contrast to "text" fields, "string" fields are stored and indexed exactly as provided, without any text analysis or transformations.
+
+This field type is suitable for fields like IDs (e.g. ORCIDs) or enums, where exact matches are required when searching.
+
+See also #11147 and #11321.
+
+### Tabular Tags Can Now Be Replaced
+
+Previously the API POST /files/{id}/metadata/tabularTags could only add new tags to the tabular tags list. Now with the query parameter ?replace=true the list of tags will be replaced.
+
+See also [the guides](https://guides.dataverse.org/en/6.7/api/native-api.html#updating-file-tabular-tags), #11292, and #11359.
+
+### Make Data Count Improvements
+
+Counter Processor, used to power Make Data Count metrics in Dataverse, is now maintained in the https://github.com/gdcc/counter-processor repository. Multiple improvements to efficiency and scalability have been made. The example `counter_daily.sh` and `counter_weekly.sh` scripts that automate using Counter Processor, available from the [MDC section](https://guides.dataverse.org/en/6.7/admin/make-data-count.html) of the Dataverse Guides have been updated to work with the latest Counter Processor release and also have minor improvements. See also #11489.
+
+### Improved Navigation for Guides
+
+Navigation across [the guides](https://guides.dataverse.org/en/6.7) has been improved. You can now click in the upper left to go "home". The navbar has been simplified with fewer links. The bottom of every page now has "Next" and "Previous" links. A "Source" link at the bottom has also been added. See also #10942.
+
+### Video Subtitles (vtt Files)
+
+Video subtitles (vtt files) are now supported and indexed using full text indexing, if [configured](https://guides.dataverse.org/en/6.7/installation/config.html#solrfulltextindexing).
+
+All new files uploaded with a .vtt extension will be assigned the context type "text/vtt" and shown as "Web Video Text Tracks". See upgrade instructions below to convert existing files.
+
+The upgrade instructions below also explain how to upgrade to v1.5 of the Dataverse Previewers, which includes and updated video previewer that supports subtitles. The new previewer version presents vtt files as subtitles for videos, and the naming convention is `<video-basename>.<language-tag>.vtt`. The previewer does not rely on the content type. A proper content type may hint users to ask permission for the subtitles together with a video.
+
+See also #11041.
+
+### Dataset Types Can Set Available Licenses
+
+Licenses (e.g. "MIT") can now be linked to dataset types (e.g. "software") using new superuser APIs. The create Dataset Type APIs have been extended to allow you to set metadata blocks and/or licenses on the creation of a Dataset Type.
```

Comment:
Is it just creation or can you edit an existing Dataset Type's mdbs and/or licenses?

Reply:
You can edit existing through separate endpoints. I added a note: 03b7032f2c